.github/workflows/tests-integration-reusable.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     runs-on: ${{ inputs.os }} 
 
     steps:
-    - uses: LouisBrunner/checks-action@v2.0.0
```

Comment:
just checking it is intentional that is being downgraded

Reply:
Good catch!! Fixed it, it wasn't intentional.